### DISCUSSION TITLE
JIT calc() option

### DISCUSSION THREAD

**Discussion by mikedejong** [tailwindlabs/tailwindcss#3986] [Help] [answered]
Hi,

Is there a possibility to use `w-[100% - 24px]` with the calc() method https://developer.mozilla.org/en-US/docs/Web/CSS/calc() instead of absolute value like `w-824px`?

**Comment by adamwathan** [comment, marked as answer]
Yeah! Try `w-[calc(100%-24px)]`, important that there's no spaces.

**Reply by halitguvenilir** [reply]
Thanks @adamwathan good job Jit is awesome :)

**Reply by dawin09** [reply]
Hello @adamwathan may I do something like: `w-[calc(w-2/3-10px)]`

**Reply by DanielBailey-web** [reply]
@dawin09 You don't need the second `w-`

Try: `w-[calc(2/3-10px)]`
